### ISSUE TITLE
VS Code notification does not stop appearing after pressing "Don't show again"

### ISSUE THREAD

**Issue by coopbri** [wallabyjs/quokka#579]
### Issue description or question
When Quokka.js is enabled in a JS/TS file and code is highlighted from an empty newline underneath some code and up, pressing `"Don't show again"` on the `"Only 'Pro' edition of Quokka.js supports Live Value Display feature."` notification popup doesn't do anything; the notification pops up again. Here is a GIF of the issue:

![7-1f8663ad16b9](https://user-images.githubusercontent.com/20056195/106378695-accc8800-639e-11eb-9a3d-ba6696a51737.gif)

### Sample code
Highlight below code + newline:

```javascript
// highlight below line + newline underneath
console.log("test");
```

### Sample repository link

N/A

### Quokka.js Console Output

N/A

### Code editor version

Visual Studio Code v1.52.1

### OS name and version

Arch Linux (kernel v5.10.11)

**Comment by ArtemGovorov** [comment]
Thanks for letting us know. The issue is fixed and the fix is published in the latest v1.0.340.

**Comment by coopbri** [comment]
Thanks so much @ArtemGovorov, can confirm this is no longer an issue. That was fast, very impressive 🙂
